src/main.js

```diff
@@ -1,6 +1,9 @@
 import Vue from 'vue'
 import App from './App.vue'
 
+import "../node_modules/ag-grid-community/dist/styles/ag-grid.css";
```

Comment:
Please move these css imports to local components.

Reply:
CSS imports moved to its local component